sechub-cli/src/mercedes-benz.com/sechub/cli/config_test.go

```diff
@@ -381,6 +382,20 @@ func Test_validateTimeoutOrWarning(t *testing.T) {
 	sechubTestUtil.AssertEquals(bigValue, result2, t)
 }
 
+func Test_validateInitialWaitIntervalOrWarning(t *testing.T) {
+	// PREPARE
+	var okay int64 = 60 * int64(time.Second) // 60s
+	var tooSmall int64 = 10000000            // 0.01s
```

Comment:
```suggestion
	var tooSmall int64 = 10_000_000            // 0.01s
```

Reply:
For my taste, this looks ugly :grimacing: I didn't know that this is valid Go code...
I changed it so that the value is now computed similar to the line before.